ext/pcntl/pcntl.c

```diff
@@ -1622,6 +1631,88 @@ PHP_FUNCTION(pcntl_getcpu)
 }
 #endif
 
+#if defined(HAVE_PTHREAD_SET_QOS_CLASS_SELF_NP)
+static qos_class_t qos_zval_to_lval(const zval *qos_obj)
+{
+	qos_class_t qos_class = QOS_CLASS_DEFAULT;
+	zend_string *entry = Z_STR_P(zend_enum_fetch_case_name(Z_OBJ_P(qos_obj)));
+
+	if (zend_string_equals_cstr(entry, "UserInteractive", sizeof("UserInteractive") - 1)) {
```

Comment:
Please use zend_string_equals_literal for these instead, it's less error-prone.

Reply:
alright